Stagehand/Classes/Testing/FBSnapshotTestCase+AnimationGIF.swift

```diff
@@ -505,30 +505,32 @@ extension FBSnapshotTestCase {
         }
 
         let firstFrameProperties = [
-            kCGImagePropertyGIFDictionary: [
-                kCGImagePropertyGIFUnclampedDelayTime: firstFrameDuration,
+            kCGImagePropertyPNGDictionary: [
+                kCGImagePropertyAPNGUnclampedDelayTime: firstFrameDuration,
             ],
         ] as CFDictionary
 
         let lastFrameProperties = [
-            kCGImagePropertyGIFDictionary: [
-                kCGImagePropertyGIFUnclampedDelayTime: lastFrameDuration,
+            kCGImagePropertyPNGDictionary: [
+                kCGImagePropertyAPNGUnclampedDelayTime: lastFrameDuration,
             ],
         ] as CFDictionary
 
         let url = URL(fileURLWithPath: NSTemporaryDirectory())
             .appendingPathComponent(UUID().uuidString)
-            .appendingPathExtension(".gif")
+            .appendingPathExtension("png")
```

Comment:
Huh, I guess the `.` in `.gif` was just getting ignored before?

Reply:
It was actually outputting a file that ended in `..gif`, which still works but is unnecessary.